### DISCUSSION TITLE
Calculate inventory and crop value

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#1467] [Ideas]
![calculateSFL](https://user-images.githubusercontent.com/74167741/193440890-14c643c8-cd08-49eb-b6c9-b1dc5dbf6dd2.png)
I don't know if it is also necessary for other users.

But I see the need to know the market value of my crops without having to calculate them one by one. With this information, in addition to comparing the price in pvp trade, I can make strategic decisions, especially during the war.

**Comment by hujikol** [comment]
i suppose this feature already been implemented sometimes ago and already live.

Link to the update FIX/Shop Sell All Modal #1431 
![image](https://user-images.githubusercontent.com/12151051/193535692-5172091f-d86f-48b2-8d0c-d4d6ba95d571.png)

**Reply by future2back** [reply]
@hujikol Thank you for participating. First idea is to bring the _calculation closer to the user interface_. Second is to _decrease the number of user actions_ if there is a need to know the value of the entire inventory. Lets invite @veenhdek and @adamhannigan for this discussion and see if we can find a better solution.

**Reply by future2back** [reply]
If i click "Sell all" to check calculation, and by mistake click on "Yes" insted of "No"?

Total calculation is necessary, but where it is, with an error, can be frustrating for the user.
